controllers/foodsController.js

```diff
@@ -30,6 +30,35 @@ const index = async (req, res) => {
   }
 }
 
+const show = async (req, res) => {
+  try {
+    const food = await Food.findOne({
+      where: {
+        id: req.params.id
+      }
+    });
+    res.setHeader("Content-Type", "application/json");
+    res.status(200).send(JSON.stringify(food));
+  } catch (error) {
+    res.setHeader("Content-Type", "application/json");
+    res.status(404).send({error});
+  }
+}
+
+const update = async (req, res) => {
+  try {
+    let food = await Food.findOne({where: {id: req.params.id}})
+    food.name = req.body.food_name
+    food.calories = req.body.calories
+    food.save({fields: ['name', 'calories']})
```

Comment:
@PeregrineReed Is this just necessary with Sequelize? What's "fields"?

Reply:
There's another way to do it with
`food.update({where: {id: req.params.id}}, {fields: ['name', 'calories']})`
that I saw. I was trying stuff out and left it this way for the push but am fine to change it back if you'd prefer.